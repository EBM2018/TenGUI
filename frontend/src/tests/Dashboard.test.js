import ActTextCheck from '../react/Dashboard/Dashboard';
import React from 'react';
import renderer from 'react-test-renderer';


test ('derp',()=>{
    expect('True').toBeTruthy;
});
