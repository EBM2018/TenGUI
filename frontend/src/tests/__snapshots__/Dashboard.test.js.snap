// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
<div
  className="bg-color"
>
  <div
    className="columns"
  >
    <div
      className="column is-3"
    >
      <h1
        className="title is-h1 add-margin-left"
      >
        Ma bibliothèque
      </h1>
    </div>
    <div
      className="column is-full"
    >
      <button
        className="button is-link"
        name="button"
        onClick={[Function]}
        type="button"
      >
        Démarrer une séance
      </button>
    </div>
  </div>
  <div
    className="columns"
  >
    <p
      className="column add-margin-left add15-margin-top is-5"
    >
      <h2
        className="is-h2"
      >
        Mes Activités
      </h2>
      <button
        className="button is-info is-inverted"
        name="button"
        onClick={[Function]}
        type="button"
      >
        Ouverture
      </button>
      <button
        className="button is-info is-inverted"
        name="button"
        onClick={[Function]}
        type="button"
      >
        Feedback
      </button>
      <h2
        className="is-h2"
      >
        Mes Actions
      </h2>
      <div
        className="column is-5 has-text-left"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted is-fullwidth"
            name="button"
            type="button"
          >
            Vous avez compris ?
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-brain"
            />
          </span>
        </p>
      </div>
      <div
        className="column is-5 has-text-left"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted is-fullwidth"
            name="button"
            type="button"
          >
            Faites un résumé ?
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-file"
            />
          </span>
        </p>
      </div>
      <div
        className="column is-5 has-text-left"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted is-fullwidth"
            name="button"
            type="button"
          >
            Vous en êtes où ?
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-clipboard-list"
            />
          </span>
        </p>
      </div>
      <div
        className="column is-5 has-text-left"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted is-fullwidth"
            name="button"
            type="button"
          >
            Reboost
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-flushed"
            />
          </span>
        </p>
      </div>
    </p>
    <p
      id="secondColumn"
    >
      <div
        className="column is-6 add-margin-right"
      >
        <h2
          className="is-h2"
        >
          Prévisualisation de l'activité
        </h2>
        <h3
          className="is-h3"
        >
          Fonctionnement
        </h3>
        <p
          className="is-body"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
        <h3
          className="is-h3"
        >
          Effet
        </h3>
        <p
          className="is-body"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
        <h3
          className="is-h3"
        >
          Affichage
        </h3>
        <p
          className="is-body"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
      </div>
    </p>
  </div>
</div>
`;
